.golangci.yml

```diff
@@ -47,6 +47,9 @@ linters:
     - usestdlibvars # using variables/constants from the standard library
     - usetesting # report function to be replace by testing
     - whitespace # unnecessary newlines
+  disable:
+    # TODO: It will be dropped when the Go version migration is done.
```

Comment:
usetesting is automatically enabled when Go 1.24 is used. This linter diagnose the calls that can be replaced by functions defined in testing package. e.g. os.Setenv -> testing.Setenv.
It is what I'm going to do after the migration to Go1.24 since the diff is very large. So I disable it once, then would tackle it later on another PR.

Reply:
Sounds good! Maybe open a small issue to track it

Reply:
I opened https://github.com/kubernetes-sigs/cluster-api/issues/12179 👍

Reply:
Thx!